src/components/InputWrapper.tsx

```diff
@@ -0,0 +1,55 @@
+import styled from "styled-components";
```

Comment:
Precisa importar o React.

```suggestion
import React from "react";
import styled from "styled-components";
```